aries-core/src/js/components/core/Tiles/stories/tileHeader.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Tile, Tiles } from 'aries-core';
+import { storiesOf } from '@storybook/react';
+
+import {
+  Box,
+  Button,
+  Footer,
+  Grommet,
+  Header,
+  Heading,
+  Paragraph,
+} from 'grommet';
+
+import { More } from 'grommet-icons';
+import { hpeFont } from './prototype';
+
+export const TileHeader = () => (
+  <Grommet theme={hpeFont} full>
+    <Tiles margin="large">
+      <Tile height="medium" width="medium">
+        <Header pad="small">
+          <Heading level={4} size="xsmall">
+            Custom Header
+          </Heading>
+          <Button icon={<More color="control" />} onClick={() => {}} />
+        </Header>
+        <Box pad="small">
+          <Paragraph>
+            This custome Tile includes custom properties of: height, width &
+            Heading. This Tile is built with the Grommet components of Header &
+            Footer.
+          </Paragraph>
+        </Box>
+        <Box flex />
+        <Footer
+          pad="medium"
+          background="light-4"
+          round={{ size: 'small', corner: 'bottom' }}
```

Comment:
If the Tile has `overflow="hidden"`, you don't need to have the Footer match the rounding of the Tile.

Reply:
Correct, this line is cleaned out on my last commit.